tests/e2e/vc_reboot_volume_lifecycle.go

```diff
@@ -185,10 +177,16 @@ var _ bool = ginkgo.Describe("Verify volume life_cycle operations works fine aft
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		err = waitForHostToBeUp(e2eVSphere.Config.Global.VCenterHostname)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
-		ginkgo.By(fmt.Sprintf("Waiting for %v for host to come up fully", VCRebootWaitTime))
-		time.Sleep(time.Duration(VCRebootWaitTime) * time.Second)
 		ginkgo.By("Done with reboot")
 
+		var wg sync.WaitGroup
+		wg.Add(4)
+		go checkVcenterServicesParallel(spsServiceName, vcAddress, svcRunningMessage, &wg)
+		go checkVcenterServicesParallel(vsanhealthServiceName, vcAddress, svcRunningMessage, &wg)
+		go checkVcenterServicesParallel(vpxdServiceName, vcAddress, svcRunningMessage, &wg)
+		go checkVcenterServicesParallel(wcpServiceName, vcAddress, svcRunningMessage, &wg)
```

Comment:
You can fetch o/p of `service-control --status` and see which services are running/stopped instead of the parallely spamming the VC with SSH calls. Today it is 4 services, tomorrow if we require to check more, then it will become an issue.

Have a method to check for VC essentials services to be up, where you parse the o/p of `service-control --status` and verify all the required services are running.